apps/system/utils/secure_log_cmd.c

```diff
@@ -0,0 +1,109 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/****************************************************************************
+ * Preprocessor Definitions
+ ****************************************************************************/
+#define LOG_SS_SLOT_PATH        "ss/32"
+
+/********************************************************************************
+ * Private Function Prototypes
+ ********************************************************************************/
+
+static void secure_log_on(int argc, char **args);
+static void secure_log_off(int argc, char **args);
+
+const static tash_cmdlist_t secure_log_cmds[] = {
+	{"seclog_on",      secure_log_on,    TASH_EXECMD_SYNC},
+	{"seclog_off",     secure_log_off,   TASH_EXECMD_SYNC},
```

Comment:
Is there any reason that the command should be separated for on and off?
How about providing a single command and giving an argument for on and off?
```
seclog on
or 
seclog off
```